src/applications/personalization/profile/components/personal-health-care-contacts/PersonalHealthCareContacts.jsx

```diff
@@ -9,18 +9,21 @@ import { focusElement } from '~/platform/utilities/ui';
 import Contacts from './Contacts';
 import Loading from './Loading';
 import LoadFail from '../alerts/LoadFail';
+import NonVAPatientMessage from './NonVAPatientMessage';
 
 const PAGE_TITLE = 'Personal Health Care Contacts | Veterans Affairs';
 
 const PersonalHealthCareContacts = ({
   fetchProfileContacts = fetchProfileContactsFn,
 }) => {
   const dispatch = useDispatch();
+  const vaPatient = useSelector(state => state?.user?.profile?.vaPatient);
```

Comment:
Suggestion: we might want to use the platform-provided selector for data. `{ isVAPatientSelector } from '~/platform/user/selectors';`

Reply:
thanks, added this in